terraform/modules/kube_pod/main.tf

```diff
@@ -108,7 +108,7 @@ locals {
   }]
 
   // Static env variables (secret)
-  common_static_secret_env = [for k in var.secrets : {
+  common_static_secret_env = [for k, v in var.secrets : {
```

Comment:
I believe `v` isnt used which is why it was omitted. Was this causing an issue?

Reply:
Big time, without v k was just the secret value, it was both borking applies if you had duplicate values for secrets and exposing said values in the state file